### ISSUE TITLE
port V_IO in A9G

### ISSUE THREAD

**Issue by shmusb1999** [Ai-Thinker-Open/GPRS_C_SDK#445]
Can we use port V_IO in A9G as a 5v/3v DC/DC regulator ?

how much ripple does it have and how is the limitation of current ?

best regards

**Comment by zkemble** [comment]
Yes, the datasheet does say that you can use it to power other 3V stuff and that the maximum current draw is 10mA. It doesn't give any ripple specs (or any other specs like accuracy), but I'm pretty sure it's a linear regulator rather than switching.
